src/plugins/contributorsPlugin/gatsby-node.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Author: Patrick Arminio <patrick.arminio@gmail.com>
+// Licensed under the MIT License (MIT), see
```

Comment:
This says MIT licensed by Patrick Arminio, but the LICENSE file says BSD Zero licensed by Gatsby, Inc.  Which is it?

Reply:
I have implemented the change for licenses to be consistent with the MIT License.